.rubocop.yml

```diff
@@ -1,2 +1,4 @@
+Style/TrailingCommaInLiteral:
```

Comment:
Opened https://github.com/voxpupuli/voxpupuli-test/pull/31 as I think that's where this file is now managed.

Reply:
This is merged, though not sure about the release process for that gem. Anyhow, this should probably stay in the commit as it will just get overwritten in the next sync and keep doing the same thing.